container.go

```diff
@@ -482,6 +482,113 @@ func (c *Client) TopContainer(id string, psArgs string) (TopResult, error) {
 	return result, nil
 }
 
+// NetworkStats describes the data related to the container's memory usage.
+type NetworkStats struct {
+	RXdropped uint64 `json:"rx_dropped"`
+	RXbytes   uint64 `json:"rx_bytes"`
+	RXerrors  uint64 `json:"rx_errors"`
+	TXpackets uint64 `json:"tx_packets"`
+	TXdropped uint64 `json:"tx_dropped"`
+	RXpackets uint64 `json:"rx_packets"`
+	TXerrors  uint64 `json:"tx_errors"`
+	TXbytes   uint64 `json:"tx_bytes"`
+}
+
+// MemoryStats describes the data realted to the container's memory usage.
+type MemoryStats struct {
+	Stats           DetailedMemoryStats `json:"stats"`
+	MaxUsage        uint64              `json:"max_usage"`
+	Usage           uint64              `json:"usage"`
+	Failcnt         uint64              `json:"failcnt"`
+	Limit           uint64              `json:"limit"`
+	PercentageInUse float64
+}
+
+// DetailedMemoryStats describes detailed data related to the container's memory usage.
+type DetailedMemoryStats struct {
+	TotalPgmajfault         uint64 `json:"total_pgmajfault"`
+	Cache                   uint64 `json:"cache"`
+	MappedFile              uint64 `json:"mapped_file"`
+	TotalInactiveFile       uint64 `json:"total_inactive_file""`
+	Pgpgout                 uint64 `json:"pgpgout"`
+	Rss                     uint64 `json:"rss"`
+	TotalMappedFile         uint64 `json:"total_mapped_file"`
+	Writeback               uint64 `json:"writeback"`
+	Unevictable             uint64 `json:"unevictable"`
+	Pgpgin                  uint64 `json:"pgpgin"`
+	TotalUnevictable        uint64 `json:"total_unevictable"`
+	Pgmajfault              uint64 `json:"pgmajfault"`
+	TotalRss                uint64 `json:"total_rss"`
+	TotalRssHuge            uint64 `json:"total_rss_huge"`
+	TotalWriteback          uint64 `json:"total_writeback"`
+	TotalInactiveAnon       uint64 `json:"total_inactive_anon"`
+	RssHuge                 uint64 `json:"rss_huge"`
+	HierarchicalMemoryLimit uint64 `json:"hierarchical_memory_limit"`
+	TotalPgfault            uint64 `json:"total_pgfault"`
+	TotalActiveFile         uint64 `json:"total_active_file"`
+	TotalPgpgout            uint64 `json:"total_pgpgout"`
+	TotalCache              uint64 `json:"total_cache"`
+	InactiveAnon            uint64 `json:"inactive_anon"`
+	ActiveFile              uint64 `json:"active_file"`
+	Pgfault                 uint64 `json:"pgfault"`
+	InactiveFile            uint64 `json:"inactive_file"`
+	TotalPgpgin             uint64 `json:"pgpgin"`
```

Comment:
Please use:

- TotalPageMajorFaults instead of TotalPgmajfault
- PageMajorFaults instead of Pgmajfault
- PageOut instead of Pgpgout
- PageIn instead of Pgpgin
- TotalRSS instead of TotalRss
- TotalRSSHuge instead of TotalRssHuge
- RSSHuge instead of RssHuge
- TotalPageFaults instead of TotalPgFault
- TotalPageOut instead of TotalPgpgout
- PageFaults instead of Pgfault
- TotalPageIn instead of TotalPgpgin